src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/stars';
+@import '/src/styles/blocks/variables';
```

Comment:
The order of imports should be as follows: variables, general styles, and then blocks. Please adjust the order of your imports accordingly.